sympy/integrals/tests/test_failing_integrals.py

```diff
@@ -177,12 +171,6 @@ def test_issue_14709a():
     # assert i == 5*h**2*pi/16
 
 
-@slow
-@XFAIL
-def test_issue_14398():
-    assert not integrate(exp(x**2)*cos(x), x).has(Integral)
```

Comment:
This doesn't seem to be passing now:
```python
In [1]: integrate(exp(x**2)*cos(x), x)
Out[1]: 
⌠                
⎮  ⎛ 2⎞          
⎮  ⎝x ⎠          
⎮ ℯ    ⋅cos(x) dx
⌡  
```

Reply:
It evaluates on my machine at commit 03fcec8a5f. If it is not evaluating then other tests in `test_manual` should fail.

```python
In [20]: integrate(exp(x**2)*cos(x), x)
Out[20]: 
    1/4     ⎛    ⅈ⎞       1/4     ⎛    ⅈ⎞
√π⋅ℯ   ⋅erfi⎜x - ─⎟   √π⋅ℯ   ⋅erfi⎜x + ─⎟
            ⎝    2⎠               ⎝    2⎠
─────────────────── + ───────────────────
         4                     4         

In [21]: 
```